app/src/androidTest/java/org/mozilla/tv/firefox/helpers/MockServerHelper.kt

```diff
@@ -24,3 +30,32 @@ object MockServerHelper {
         return uris
     }
 }
+
+/**
+ * A [MockWebServer] [Dispatcher] that will return Android assets in the body of requests.
+ *
+ * If the dispatcher is unable to read a requested asset, it will fail the test by throwing an
+ * Exception on the main thread.
+ *
+ * @sample [org.mozilla.tv.firefox.ui.BasicNavigationTest.whatever]
```

Comment:
Was `.whatever` a placeholder that should be updated?

Reply:
This is fixed in a later commit.